<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="textDiscription2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>1.Cгенерированная последовательность будет записана в файл &quot;PRnumbers.txt&quot; в каталоге проекта.
2.Если файл не пуст, то он будет перезаписан.</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGeneratedParams">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерированные параметры:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputLength">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>381</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите желаемую длину последовательности ПСЧ:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputSeqLength">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputSeq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>501</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGenerateSeq">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Сгенерировать
последовательность 
ПСЧ</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textDiscription1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>1.В качестве ГПСЧ  используется линейный конгруэнтный
генератор, использующий рекуррентное соотношение.
2.формула генерации:
ci = ( a * ci-1 + b) (mod m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputSeq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерированная последовательность:</string>
    </property>
   </widget>
   <widget class="QLabel" name="laberDiscription2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Примечание:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDiscription1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Примечание:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGenerateParams">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать
параметры
для ГПСЧ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonViewDiagram">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>580</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Посмотреть гистограмму</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputGeneratedParams">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
